recipes/samtools/1.9+37-g5708485/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+n=`expr $CPU_COUNT / 4 \| 1`
+
+autoreconf
```

Comment:
Do we need to apply here a fix from https://github.com/wtsi-npg/npg_conda/pull/85 ?

Reply:
No, samtools does not have any generated files checked in to git.